scripts/autolabels.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+
+ : <<'BASH_MODULE_DOCS'
+
+This file contains the dictionary for converting a modified dir into a github label,
+as well as the script to perform the conversion.
+
+Rules:
+* only lines starting with `|t-` are considered;
+* repeated spaces, commas and pipes (|) are ignored;
+* `Mathlib` is added by default to each `Root folder`.
+
+Exception: `Tactic/Linter` is converted into `Tactic-Linter` internally, to simplify separating it from
+`Tactic/anything_other_than_linter`.
+
+|Label                   |Root folders                                                                       |
+|-                       |-                                                                                  |
+|t-algebra               | Algebra, FieldTheory, RingTheory, GroupTheory, RepresentationTheory, LinearAlgebra|
+|t-algebraic-geometry    | AlgebraicGeometry, Geometry/RingedSpace                                           |
+|t-euclidean-geometry    | Geometry.Euclidean                                                                |
+|t-differential-geometry | Geometry/Manifold                                                                 |
+|t-analysis              | Analysis                                                                          |
+|t-category-theory       | CategoryTheory                                                                    |
+|t-combinatorics         | Combinatorics                                                                     |
+|t-computability         | Computability                                                                     |
+|t-condensed             | Condensed                                                                         |
+|t-data                  | Data                                                                              |
+|t-dynamics              | Dynamics                                                                          |
+|t-linter                | Tactic/Linter                                                                     |
+|t-logic                 | Logic, ModelTheory                                                                |
+|t-measure-probability   | MeasureTheory, Probability, InformationTheory                                     |
+|t-meta                  | Tactic, Lean, Init, Util, Mathport, Control, Testing                              |
+|t-number-theory         | NumberTheory                                                                      |
+|t-order                 | Order                                                                             |
+|t-topology              | Topology, AlgebraicTopology                                                       |
+|t-set-theory            | SetTheory                                                                         |
+
+BASH_MODULE_DOCS
+
+# strips replaces repeated `|, ` with a single space, then only keeps lines beginning with `t-`
```

Comment:
This sentence does not parse. Do you mean to delete the word "strips"?

Reply:
Removed `strips`, thanks!